osc-service-api/src/main/java/org/osc/core/broker/service/dto/SecurityGroupMemberItemDto.java

```diff
@@ -118,4 +123,38 @@ public void setParentOpenStackId(String parentOpenStackId) {
         this.parentOpenStackId = parentOpenStackId;
     }
 
+    public Set<VMPortDto> getVmPorts() {
+        return this.vmPorts;
+    }
+
+    public void setVmPorts(Set<VMPortDto> vmPorts) {
+        this.vmPorts = vmPorts;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
use eclipse generated toString method unless we need something specific. less code to maintain. same for other file